auth/credentials/internal/externalaccount/externalaccount.go

```diff
@@ -399,7 +400,7 @@ func newSubjectTokenProvider(o *Options) (subjectTokenProvider, error) {
 
 func getGoogHeaderValue(conf *Options, p subjectTokenProvider) string {
 	return fmt.Sprintf("gl-go/%s auth/%s google-byoid-sdk source/%s sa-impersonation/%t config-lifetime/%t",
-		goVersion(),
+		header.GoVersion(),
```

Comment:
the unknown value below this can be known now.

Reply:
Fixed, thanks!